components/_patterns/00-utilities/_variables.scss

```diff
@@ -1,5 +1,12 @@
 //
-// BASE COLORS
+// ABOUT THIS FILE:
+// This file serves as a home for all your variables and also acts as a
+// configuration file for certain Bear Skin behaviors.
+//
+
+//
+// BEAR SKIN COLORS
+// Default colors that come packaged with Bear Skin.
 //
 $black:              hsla(0, 0%, 0%, 1);
```

Comment:
We should capitalize hsla - HSLA(0, 0%, 0%, 1) - or just switch back to using hex colors. The lower case hsla() is a Sass function that converts back to hex.

Reply:
@tortillaj I'm in favor of moving everything to hex. I like being able to use Sass color functions (lighten, darken, transparentize, etc.), and they don't work with HSLA values.

Reply:
Either is fine by me. But let's see what @alioso thinks too.

Of course you don't need the Sass functions for HSLA, that was partly the point of using HSLA. Need to lighten or darken a color? Change the L. Transparentize? Change the A.

Reply:
@tortillaj I meant that I prefer using the Sass functions to copying a color variable's value and tweaking it. Sass functions make the relationship to the original color clear.

However, this is only really an issue if I don't create a new variable for the altered color. Now that I'm thinking about it, I might prefer to do things in a way that encourages creating new variables to using one-off color values.

Reply:
Exactly my point. I'm hoping that we create light and dark color variables and reuse them.

Reply:
@jkaeser just convert colors to hex